croptalk/create_doc_data.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
+from sqlalchemy import create_engine
+from dotenv import load_dotenv
+import os
+
+from croptalk.utils import query_sql_data
+
+load_dotenv("secrets/.env.secret")
+
+# Define your database connection URL
+db_url = os.environ["POSTGRES_URI"]
+
+# Create the SQLAlchemy engine
+engine = create_engine(db_url)
+
+if __name__ == '__main__':
+    # TODO this should be a SQL statement using the data from the generated weaviate_collection
```

Comment:
Does the TODO apply to the next line? I am not entirely sure...

And that's not something we want to address now... or is it?

Reply:
Yes it applies. It would be great if everything was done in a seemless manner. Now it seems like the scope of the db will change so there is a risk in connecting it to the whole pipeline.